artifactory/commands/transferfiles/transfer.go

```diff
@@ -309,7 +311,7 @@ func (tdc *TransferFilesCommand) initCurThreads(buildInfoRepo bool) error {
 		}
 	}
 
-	log.Info("Running with " + strconv.Itoa(curThreads) + " threads...")
+	log.Info("Running with maximum " + strconv.Itoa(curThreads) + " working threads...")
```

Comment:
You can use multiple arguments when calling `log.Info`, which makes it look better (no need to the separator white-spaces and string concatenation) - 
```
log.Info("Running with maximum", strconv.Itoa(curThreads), "working threads...")
```